deploy/alerts/README.md

```diff
@@ -0,0 +1,23 @@
+# Predefined alerts
+
+These alarming rules come in two files described in the following
+
+## alerts.yaml
+These alarms are triggered by a substantial change (over a certain
+amount of standard deviation). This requires a bit of preparation
```

Comment:
"requires a bit of preparation" this statement indicates some pre work needs to be done but doesnt quite say what it is.  Should we just drop this line or make it a bit less ambiguous.

Reply:
You're right, this needs a bit more explanation. In the way we are doing this here, it requires about 4 rules to get created per alarm for a single metric.